portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsColorField.testcase

```diff
@@ -120,13 +120,44 @@ definition {
 	}
 
 	@description = "Verify that a help text can be added to a Color Field"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test HelpTextCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133142 HelpTextCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Color");
+
+		FormViewBuilder.setFieldHelpText(fieldLabel = "Color Help Text");
+	
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Color",
```

Comment:
@leilanyaragao, isn't it the `fieldLabel` instead of the `fieldName`?

Can we please rename it accordingly?

Reply:
@leilanyaragao, please see me comment above. :wink: